radix-engine-interface/src/blueprints/epoch_manager/invocations.rs

```diff
@@ -55,6 +55,11 @@ impl EpochManagerInitialConfiguration {
         self.min_validator_reliability = new_value;
         self
     }
+
+    pub fn with_num_owner_stake_units_unlock_epochs(mut self, new_value: u64) -> Self {
```

Comment:
I am worried that then David will come and say "let's have long, descriptive names, bring back the `with_num_owner_stake_units_unlock_epochs()`" :P

The field is named `num_owner_stake_units_unlock_epochs` so the "update" method kinda has to be `with_X()`. I prefer to keep the `_epochs` suffix there, since it specifies the "time units" of that unlock delay. (normally I would agree to drop the unit from the name, IF we had a well-named data type representing `epochs`, but currently we use the generic u64)